### PR TITLE
Prozzie config command refactor

### PR DESCRIPTION
**Pull request by Chorro**
I've refactored prozzie config command in order to improve the logic and have clearer code.

### CODE REVIEW

File: tests/tests_config01.bash [hunk 1]
```diff
@@ -241,30 +205,24 @@ testDescribeMustShowHelpIfModuleIsNotPresent() {
...
 testConfigMustShowHelpIfTryToSetMqttAndSyslogModules() {
-    if "${PROZZIE_PREFIX}"/bin/prozzie config mqtt kafka.topic myTopic; then
-        ${_FAIL_} '"prozzie config mqtt kafka.topic myTopic must show help"'
+    if "${PROZZIE_PREFIX}"/bin/prozzie config set mqtt kafka.topic=myTopic; then
+        ${_FAIL_} '"prozzie config set mqtt kafka.topic=myTopic must show help"'
     fi
-    if "${PROZZIE_PREFIX}"/bin/prozzie config syslog kafka.topic myTopic; then
-        ${_FAIL_} '"prozzie config syslog kafka.topic myTopic must show help"'
+    if "${PROZZIE_PREFIX}"/bin/prozzie config set syslog kafka.topic=myTopic; then
+        ${_FAIL_} '"prozzie config ste syslog kafka.topic=myTopic must show help"'
```
Comment (eugpermar) [inline tests/tests_config01.bash]: ste -> set

File: cli/include/config.bash [hunk 2]
```diff
@@ -213,17 +213,29 @@ zz_get_var() {
...
 zz_get_vars () {
-        declare -A env_content
+        # Get env file
+        declare -r env_file="$1"
+        shift
 
-        while IFS='=' read -r key val || [[ -n "$key" ]]; do
+        # If keys list is empty then show all variables
+        if [[ -z $@ ]]; then
+            declare -A env_content
```
Comment (eugpermar) [inline cli/include/config.bash]: This branch needs a test

File: cli/include/config.bash [hunk 3]
```diff
@@ -242,25 +254,50 @@ zz_get_vars () {
...
 
-            printf -v new_value "%s=%s" "$2" "$value"
+        printf -v new_value "%s=%s" "$2" "$value"
 
-            sed -i '/'"$2"'.*/c\'"$new_value" "$1"
-        else
-            printf "Variable '%s' not recognized! No changes made to %s\n" "$2" "$1" >&2
+        sed -i '/'"$2"'.*/c\'"$new_value" "$1"
+    else
+        printf "Variable '%s' not recognized! No changes made to %s\n" "$2" "$1" >&2
```
Comment (eugpermar) [inline cli/include/config.bash]: This branch needs to be tested

File: cli/include/config.bash [hunk 4]
```diff
@@ -242,25 +254,50 @@ zz_get_vars () {
...
+    shift
+
+    for pair in $@; do
+        # Check if a pair of key-value is valid
+        if [[ $pair == *=* ]]; then
+            key=${pair%%=*}
+            val="${pair#*=}"
+            zz_set_var "$env_file" "$key" "$val"
+        else
+            printf "The argument '%s' isn't a valid key-value pair and won't be applied\n" "$pair" >&2
```
Comment (eugpermar) [inline cli/include/config.bash]: This branch needs to be tested

File: cli/prozzie-config.bash [hunk 5]
```diff
@@ -43,63 +45,144 @@ printHelp() {
...
+            declare env_file="$PROZZIE_ENVS/$module.env"
+
+            # If module is referred to base module then set $env_file to $base_env_file
+            if [[ "$module" =~ ^base$ ]]; then
+                env_file="$base_env_file"
+            fi
+
+            # Check if env file exists (excluded mqtt and syslog)
+            if [[ ! "$module" =~ ^(mqtt|syslog)$ && ! -f "$env_file" ]]; then
+                printf "Module '%s' does not have a defined configuration (*.env file)\n" "$module">&2
```
Comment (eugpermar) [inline cli/prozzie-config.bash]: This branch needs to be tested

File: cli/prozzie-config.bash [hunk 6]
```diff
@@ -43,63 +45,144 @@ printHelp() {
...
+                printf "You can set '%s' module configuration using setup action.\n" "$module">&2
+                printf "For more information see the command help\n" >&2
+                exit 1
+            fi
+
+            declare -r module_config_file="$PROZZIE_CLI_CONFIG/$module.bash"
+            # Check that module's config file exists
+            if [[ ! -f "$module_config_file" ]]; then
+                printf "Unknow module: '%s'\n" "$module" >&2
```
Comment (eugpermar) [inline cli/prozzie-config.bash]: This branch needs to be tested

File: tests/tests_config01.bash (hunk 1 above)
Comment (Chorro) [reply tests/tests_config01.bash]: Done 👍

File: cli/include/config.bash (hunk 3 above)
Comment (Chorro) [reply cli/include/config.bash]: Please, review (testSetWrongVariable)[https://github.com/wizzie-io/prozzie/pull/43/files#diff-077fb1ccf4e716f238eaf2ca49d08414R214]

File: cli/prozzie-config.bash (hunk 5 above)
Comment (Chorro) [reply cli/prozzie-config.bash]: Please, check [testGetWrongModuleConfiguration](https://github.com/wizzie-io/prozzie/pull/43/files#diff-077fb1ccf4e716f238eaf2ca49d08414R224)

File: cli/prozzie-config.bash (hunk 6 above)
Comment (Chorro) [reply cli/prozzie-config.bash]: Please, check [testWrongModule](https://github.com/wizzie-io/prozzie/pull/43/files#diff-077fb1ccf4e716f238eaf2ca49d08414R230)

**Pull request merged**